attest/core/src/error.rs

```diff
@@ -239,9 +239,9 @@ pub enum QuoteError {
     InvalidUtf8,
 }
 
-impl From<ConvertError> for QuoteError {
-    fn from(src: ConvertError) -> Self {
-        QuoteError::Encoding(src.into())
+impl From<base64::DecodeError> for QuoteError{ 
+    fn from(_src: base64::DecodeError) -> Self { 
+        QuoteError::Encoding(EncodingError::InvalidInput)
```

Comment:
There's an `InvalidLength` variant of `base64::DecodeError`, so we should map that to the corresponding `EncodingError` variant
```suggestion
impl From<base64::DecodeError> for QuoteError {
    fn from(src: base64::DecodeError) -> Self {
        use base64::DecodeError::*;
        let kind = match src {
            InvalidLength => EncodingError::InvalidInputLength,
            InvalidByte(_, _)|InvalidLastSymbol(_, _) => EncodingError::InvalidInput,
        };
        QuoteError::Encoding(kind)
```